src/components/CategoriesContainer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function CategoriesContainer() {
+  return (
+    <div className="container">
+      <h2>Categories</h2>
+      <p>Under Contruction</p>
+    </div>
+  );
+}
+
+export default CategoriesContainer;
```

Comment:
- [ ] Good job on implementing displaying `Under Construction` on `categories` page. But I think it would be better if you also use `redux` as you did for books. Please take a look at the screenshot of the project requirements below 👇 
![Screenshot (241)](https://user-images.githubusercontent.com/67087664/160376438-d923f059-32f1-40e1-ba47-84210e4737d2.png)